src/std/net/socks/interface.ss

```diff
@@ -0,0 +1,25 @@
+;;; -*- Gerbil -*-
+;;; © vyzo
+;;; SOCKS v4/4a/5 interface
+(import :std/interface
+        :std/contract
+        :std/io/interface)
+(export #t)
+
+;; SOCKS client interface
+(interface SOCKS
+  ;; protocol version used by the proxy
+  ;; :- symbol?
+  (protocol)
```

Comment:
Is it possible to define exactly which symbols are accepted in the contract here? Ie. something like:
```scheme
(protocol :~ (or ('SOCKS4 'SOCKS4a 'SOCKS5)))
...
```
Syntax is psudocode, but should get the idea across.

Reply:
It doesnt accept anything, it returns.

I guess we could when we have return type annotation, otherwise it is in the docs.

Something like `(one-of ...)` would do.

Reply:
I think I will add a commented `one-of`

Reply:
I was more curious than thinking it's something we should add here